src/sentry/static/sentry/app/views/settings/components/dataScrubbing/modals/localStorage.tsx

```diff
@@ -0,0 +1,44 @@
+import * as Sentry from '@sentry/browser';
+
+import localStorage from 'app/utils/localStorage';
+
+import {SourceSuggestion} from '../types';
+
+const ADVANCED_DATA_SCRUBBING_LOCALSTORAGE_KEY = 'advanced-data-scrubbing';
+
+type StorageValue = {
+  eventId: string;
+  sourceSuggestions: Array<SourceSuggestion>;
+};
+
+// TODO(Priscila): add the method below in app/utils
+const fetchFromStorage = (): StorageValue | undefined => {
+  const storage = localStorage.getItem(ADVANCED_DATA_SCRUBBING_LOCALSTORAGE_KEY);
+  if (!storage) {
+    return undefined;
+  }
+
+  try {
+    return JSON.parse(storage);
+  } catch (err) {
+    Sentry.withScope(scope => {
+      scope.setExtra('storage', storage);
+      Sentry.captureException(err);
+    });
+    return undefined;
+  }
+};
+
+const saveToStorage = (obj: StorageValue) => {
+  try {
+    localStorage.setItem(ADVANCED_DATA_SCRUBBING_LOCALSTORAGE_KEY, JSON.stringify(obj));
+  } catch (err) {
+    Sentry.captureException(err);
+    Sentry.withScope(scope => {
+      scope.setExtra('storage', obj);
+      Sentry.captureException(err);
+    });
+  }
+};
+
+export {fetchFromStorage, saveToStorage};
```

Comment:
What are we saving to local storage? Are there size concerns? When do we prune localStorage?

Reply:
we are saving the event id... we should not worry about the size... it's usually not that long.. I don't prune the localSorage, the user can change routes and when he/she/they will be back the eventId will still be available